GUIDELINES.md

```diff
@@ -171,6 +171,32 @@ Example:
 }
 ```
 
+### Request Nullable/Optional properties
```

Comment:
PP : I think you're making this very difficult to understand.

The general principle is the following.

If you don't provide a certain property in your request, the previous value will remain.
If you provide a certain property in your request, the provided value will overwrite the previous value for that property.

Hence sending `null` as a value, will empty the property.
Hence for sending a collection or an object, you need provide all items/properties.

Also: shouldn't this be in the html documentation instead of the contribution guidelines?

Reply:
The main purpose for this documentation section was to make it clear when to use `nullable` and `optional` which are very related to the design of the API and not the usage. So a section explaining this in guidelines makes a lot of sense.

How we handle empty values is something different and has its place in the general documentation indeed.